.github/ISSUE_TEMPLATE.md

```diff
@@ -1,8 +1,9 @@
 <!-- Check the following before filing an issue -->
 > Before filing an issue please check that the issue is not already addressed by the following:
 >  * [Video Guides](https://www.twilio.com/docs/api/video)
->  * [Github Issues](https://github.com/twilio/video-quickstart-swift/issues)
->  * [Changelog](https://www.twilio.com/docs/api/video/changelogs/ios)
+>  * [Changelog](https://www.twilio.com/docs/api/video/changelog-twilio-video-ios-version-2x).
+> Issues filed to this repo should be specific to the QuickStart apps, for SDK level inquiries please use: 
+>  * [Video SDK - Github Issues](https://github.com/twilio/twilio-video-ios/issues) instead.
```

Comment:
I think I would like some text that says something like: If this is an issue with the quickstart itself, file it here, if it is an issue with the SDK or how to do something with the SDK, file it there.